src/Components/SearchBox.tsx

```diff
@@ -39,7 +39,7 @@ export default class SearchBox extends React.Component {
           this.headlessSearchBox.submit();
         }}
         options={this.state.suggestions.map(
-          (suggestion: any) => suggestion.value
+          (suggestion: any) => suggestion.rawValue
```

Comment:
There is the `highlightedValue` property on the suggestion which would be more interesting to show-case. Also, it would be good to remove `any`, to get access to intellisense, and allow typescript to alert us when there is a breaking change.

Reply:
With `highlightedValue`, the part of the suggestion that matches, does not match, or is a corrected typo, can be highlighted using the controller's options. Check out the suggestions when typing into the search boxes below:

- https://static.cloud.coveo.com/searchui/v2.10081/index.html#t=All&sort=relevancy
- https://www.google.com/